pkg/build/strategies/sti/postexecutorstep.go

```diff
@@ -211,46 +219,10 @@ func (step *downloadFilesFromBuilderImageStep) execute(ctx *postExecutorStepCont
 }
 
 func (step *downloadFilesFromBuilderImageStep) downloadAndExtractFile(artifactPath, artifactsDir, containerID string) error {
```

Comment:
I don't understand the point of this function.  Did you mean for it to return the error if the err != nil?